web-app/django/VIM/apps/instruments/management/commands/import_instruments.py

```diff
@@ -15,7 +15,7 @@ class Command(BaseCommand):
 
     NOTE: For now, this script only imports instrument names in English and French. It
     also only imports a set of previously-curated instruments that have images available.
-    This list of instruments is stored in startup_data/vim_instruments_with_images-15sept.csv
+    This list of instruments is stored in startup_data/all_instruments_with_16aug_2024.csv
```

Comment:
No longer the right file name. You could probably just take out this comment and make the file name a command-line argument or a constant.